webpack.test.standalone.config.js

```diff
@@ -0,0 +1,17 @@
+const options = {
+    ts: {
+        transpileOnly: true
+    },
+    preprocessor: {
+        variables: {
+            mode: 'eager'
+        },
+        definitions: ['TEST', 'TEST_COPY_PASTE']
+    },
+    mode: 'development'
+};
+
+let config = require('./.config/webpack/base.js')(options);
+delete config.plugins;
```

Comment:
There's no real need for lazy loading and no `dash path` to resolve when running the standalone version of the table.